flavors/benchmark/aws_org.go

```diff
@@ -33,29 +34,31 @@ import (
 	"github.com/elastic/cloudbeat/resources/fetching"
 	"github.com/elastic/cloudbeat/resources/fetching/factory"
 	"github.com/elastic/cloudbeat/resources/fetching/registry"
+	"github.com/elastic/cloudbeat/resources/providers/awslib"
 )
 
-type AWSOrg struct{}
+type AWSOrg struct {
```

Comment:
```suggestion
type AwsOrg struct {
```

Reply:
I wonder what we should do for consistency with these names, the recommendation is to use all caps for acronyms: https://stackoverflow.com/a/63355758

Reply:
Name was not really introduced in this PR so we can rename it on a different PR when we decide

Reply:
I see, so we can keep it the way it's.